programmers/자릿수-더하기.js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+  var answer = 0;
+  for (; n > 0; n = Math.floor(n / 10)) {
+    answer += n % 10;
+  }
+  return answer;
+}
```

Comment:
이야... 신박하네요 참 ㅋㅋㅋㅋ
너무 창의적이었지만 가독성에는 살짝 무리가 있는 코드에요.
이렇게 짜는것보다는 ```while```문을 사용하는게 좋겠어요 ^^

```suggestion
function solution(n) {
    let answer = 0;
    while (n > 0) {
        answer += n % 10;
        n = Math.floor(n / 10);
    }
    return answer;
}
```

훨씬 가독성있지 않나요? :D